### ISSUE TITLE
About license

### ISSUE THREAD

**Issue by s4il3sh** [ubsuny/23-Homework4G4#44]
Did you guys check the availability of the license?

**Comment by pratibha77118** [comment]
Ya I am wondering how to know if I am allowed to use the liscence.

**Comment by s4il3sh** [comment]
We tried adding license and it allowed us to use it. Maybe that's not the way to check license. Let us know if you know how to check it.

**Comment by reshnashrestha** [comment]
You can see the **Terms and Conditions**

**Comment by s4il3sh** [comment]
> You can see the **Terms and Conditions**

Right! Easier would be copy the original license file and upload it to your repo.
